### PR TITLE
Added description for minimum/maximum

### PR DESCRIPTION
**Pull request by spacemanspiff2007**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17440

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/counter.markdown [hunk 1]
```diff
@@ -41,6 +41,16 @@ counter:
       required: false
       type: integer
       default: 0
+    minimum:
+      description: Minimum value the counter will have
+      required: false
+      type: integer
+      default: None
```
Comment (frenck) [inline source/_components/counter.markdown]: If it has no default, please leave it out.

File: source/_components/counter.markdown [hunk 2]
```diff
@@ -41,6 +41,16 @@ counter:
...
+    maximum:
+      description: Maximum value the counter will have
+      required: false
+      type: integer
+      default: None
```
Comment (frenck) [inline source/_components/counter.markdown]: If it has no default, please leave it out.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by spacemanspiff2007 — COMMENTED** [review]
removed default: None

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for updating the PR @spacemanspiff2007! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by spacemanspiff2007** [comment]
Added a service which can change minimum, maximum and step

**Review by frenck — APPROVED** [review]

**Pull request closed without merging**